src/invidious/routes/api/manifest.cr

```diff
@@ -126,6 +127,62 @@ module Invidious::Routes::API::Manifest
 
             i += 1
           end
+
+          if thumbnails
+            # https://dashif.org/docs/DASH-IF-IOP-v4.3.pdf
+            {"image/jpeg"}.each do |mime_type|
+              storyboards = video.storyboards
+              next if storyboards.empty?
+
+              storyboards.each do |storyboard|
+                interval = storyboard[:interval].to_i
+                # interval can be 0 https://github.com/iv-org/invidious/issues/3441
+                next if interval == 0
+
+                storyboard_height = storyboard[:storyboard_height]
+
+                # BUG: storyboardHeight can be wrong https://github.com/iv-org/invidious/issues/3440
+                # this fix/assumption is only if we have storyboard[:storyboard_count] = 1
```

Comment:
Based on discussion in https://github.com/iv-org/invidious/issues/3440, this is not the right assumption.
We need to "calculate" the image size based on which page we're on, storyboard count, and tile size